lido_sdk/network/type.py

```diff
@@ -36,4 +38,7 @@ class Network(IntEnum):
         "0x73c72beecbd832c9ce342e61a772c8cfe6f1c6d661b19a98317b5dac05ce9685",
     ],
     Network.Rinkeby: [],
+    Network.Kintsugi: [
+        ""
```

Comment:
It seems to be worth leaving an empty array, the actual WC will be taken from the network

Reply:
yeah, removed WC